pkg/controller/quota/quota_manager_interface.go

```diff
@@ -23,4 +23,5 @@ import (
 type QuotaManagerInterface interface {
 	Fits(aw *arbv1.AppWrapper, resources *clusterstateapi.Resource, proposedPremptions []*arbv1.AppWrapper) (bool, []*arbv1.AppWrapper, string)
 	Release(aw *arbv1.AppWrapper) bool
+	GetTreeNames() []string
```

Comment:
`GetTreeNames` is not a good name IMHO for this method. What do you think of this name `GetValidQuotaLabels` ?

Reply:
Done